src/GraphQL/Attributes/DefaultAstValueAttribute.cs

```diff
@@ -0,0 +1,43 @@
+using GraphQL.Types;
+using GraphQL.Utilities;
+using GraphQLParser;
+using GraphQLParser.AST;
+
+namespace GraphQL;
+
+/// <summary>
+/// Specifies the default value for an argument or a field of an input object.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter)]
+public class DefaultAstValueAttribute : GraphQLAttribute
+{
+    private readonly GraphQLValue _value;
+    private static readonly ParserOptions _options = new() { Ignore = IgnoreOptions.All };
+
+    /// <inheritdoc cref="DefaultAstValueAttribute"/>
+    public DefaultAstValueAttribute(string astValue)
+    {
+        _value = Parser.Parse<GraphQLValue>(astValue, _options);
```

Comment:
I can try doing this in analyzer to verify the correctness of the value. But this may be challenging because I need to reference the same version as the analyzed project...

Reply:
If you like. While it would be nice, I don’t think it’s necessary.